### PR TITLE
Base model objects and formulas

### PR DESCRIPTION
**Pull request by hoisinberg**
Woo

### CODE REVIEW

File: server/formulas/nutrition.formulas.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+export function getTdee(exerciseFrequency, weight, height, age, sex) {
+  if (!(exerciseFrequency in FREQUENCY_MULTIPLIER)) {
+    throw new Error(`Unrecognized exercise frequency ${exerciseFrequency}.`);
+  }
+
+  let ree;
+
+  if (sex === 'male') {
+    ree = 10 * weight.get('kg') + 6.25 * height.get('cm') - 5 * age + 5;
```
Comment (itstang) [inline server/formulas/nutrition.formulas.js:20]: I'm actually a little skeptical of these formulas, these seem a little _too_ similar

File: server/formulas/measurement.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const MASS = {
+  kg: 1,
+  lb: 2.20462
```
Comment (itstang) [inline server/formulas/measurement.js:3]: maybe we should designate how many sig figs we want. 5 sig figs seems a little unnecessary if everything else is only 2 or less

**Review by itstang — COMMENTED** [review]
Overall looks good, just a couple concerns

File: server/formulas/measurement.js (hunk 2 above)
Comment (hoisinberg) [reply server/formulas/measurement.js:3]: Hmm I'd say leave the calculations as precise as possible, and we can always control the formatting when we actually display the values. That way, we're at least not introducing error.

File: server/formulas/nutrition.formulas.js (hunk 1 above)
Comment (hoisinberg) [reply server/formulas/nutrition.formulas.js:20]: What do you propose we do?

Comment (itstang) [reply server/formulas/nutrition.formulas.js:20]: alright according to https://healthyeater.com/how-to-calculate-your-macros this is actually correct?

**Review by itstang — APPROVED** [review]

**Pull request merged**